.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,12 @@
+name: Auto approve
+
+on: pull_request_target
+
+jobs:
+  auto-approve:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: hmarr/auto-approve-action@v2.1.0
+        if: github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
+        with:
+          github-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
I took this file from our other public repo: field-editors https://github.com/contentful/field-editors/blob/master/.github/workflows/auto-approve.yml, I'm puzzled though why it works, because there's also no `GITHUB_TOKEN` in the repo secrets in field-editors

Reply:
>  no GITHUB_TOKEN in the repo secrets in field-editors

`GITHUB_TOKEN` is added GitHub at runtime with some default permissions that are enough for approving

But the `hmarr/auto-approve-action` only approves and doesn't command Dependabot to merge it which is less than ideal.

We can do that in another PR btw. This is fine for now

Reply:
Ahh I see what you mean, TIL